src/css/controls/imports/icons.less

```diff
@@ -68,6 +68,24 @@
     }
 }
 
+.jw-icon-pip {
+    .jw-svg-icon-pip-off {
+        display: none;
+
+        .jw-off& {
+            display: block;
+        }
```

Comment:
```suggestion
.jw-icon-pip {
      &.jw-on {
          display: block;
      }
    .jw-svg-icon-pip-off {
        display: none;
```

Reply:
There are two svgs for `jw-icon-pip`, one for when it is open and one for when it is closed, just like for fullscreen. Please see the code above these lines as they follow the same pattern: https://github.com/jwplayer/jwplayer/pull/3891/files#diff-8133bd0ba407ee0e937fc4b7e74b58a5d0c0de6f51134992a79e0645b9a6dd9cR53